sdk/program/src/instruction.rs

```diff
@@ -744,42 +744,3 @@ pub fn get_stack_height() -> usize {
         crate::program_stubs::sol_get_stack_height() as usize
     }
 }
-
-#[test]
-fn test_account_meta_layout() {
```

Comment:
cc/ @Lichtso

test seems dubious? basically asks, "is `repr(C)` the same as rust layout," which is a losing question

Reply:
The test was introduced in #22859 for the syscall `SyscallGetProcessedSiblingInstruction`. Later in #30027 the `repr(C)` was added to `AccountMeta` for Firedancer. So yes, we likely don't need the test anymore.